sql/moz-fx-data-shared-prod/accounts_backend_derived/monitoring_db_recovery_phones_counts_v1/metadata.yaml

```
@@ -0,0 +1,17 @@
+friendly_name: FxA DB Counts Monitoring: Recovery Phones Segmentation
```

In YAML you can't have `: ` within a plain scalar, which is why the `verify-format-yaml` CI is [failing](https://app.circleci.com/pipelines/github/mozilla/bigquery-etl/46883/workflows/321fa970-600a-4930-be8d-0ef9fe79b0a1/jobs/548245):
```
./sql/moz-fx-data-shared-prod/accounts_backend_derived/monitoring_db_recovery_phones_counts_v1/metadata.yaml
  1:40      error    syntax error: mapping values are not allowed here (syntax)
```

This needs to be quoted or use block scalar syntax.
```suggestion
friendly_name: 'FxA DB Counts Monitoring: Recovery Phones Segmentation'
```